.ci/pytorch/build.sh

```diff
@@ -6,6 +6,11 @@ set -ex
 # (This is set by default in the Docker images we build, so you don't
 # need to set it yourself.
 
+if [[ "$BUILD_ENVIRONMENT" == linux-jammy-py3.8-gcc11* ]] ; then
```

Comment:
Instead of checking for BUILD_ENVIRONMENT can we perhaps check if its anaconda kernel5+ ?

Reply:
unfortunately no, we would need to check the combination build + test for it to work.